src/app/services/ccd-config/ccd-case.config.spec.ts

```diff
@@ -160,7 +160,7 @@ describe('AppConfiguration', () => {
   }));
 
   it('should have getDocumentSecureMode return value', inject([AppConfig], (service: AppConfig) => {
-    expect(service.getDocumentSecureMode()).toBe(true);
+    expect(service.getDocumentSecureMode()).toBe(false);
```

Comment:
I've not looked into why this behaviour is different, should be reverted - just pushed with this since it's the only unit test which fails and wanted to verify integration tests on the pipeline.

Reply:
Looking further, the default value is `false` so this is actually the correct expectation.

Reply:
Looking further, the default value is `false` so this is actually the correct expectation. The fact it was passing before with `true` is probably a symptom of the issue that #3910 is looking to fix!